Code/Framework/AzToolsFramework/Tests/Prefab/Overrides/PrefabInspectorOverrideTests.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzToolsFramework/Prefab/DocumentPropertyEditor/PrefabPropertyEditorNodes.h>
+#include <AzToolsFramework/UI/PropertyEditor/ComponentEditor.hxx>
+#include <Prefab/Overrides/PrefabInspectorOverrideTestFixture.h>
+
+namespace UnitTest
+{
+    using PrefabInspectorOverrideTest = PrefabInspectorOverrideTestFixture;
+
+    // These paths depends on multiple factors like the data in the component, how its reflected to serialize and edit contexts,
+    // how different DPE adapters likes ReflectionAdapter and PrefabAdapter construct the DPE DOM etc. Therefore, these may
+    // change in the future if the data stored in the DPE DOM itself changes and need to be modified accordingly to prevent test failures.
+    constexpr AZStd::string_view domPathToTranslateProperty = "/0/3/2";
+    constexpr AZStd::string_view domPathToOverriddenTranslateProperty = "/0/3/3";
```

Comment:
I think we can add a comment about the what the path looks like for each DOM path, e.g. `/Transform Data/Translate/0`.